user/nmap/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "nmap"
+pkgver = "7.95"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--with-openssl",
+    "--without-zenmap",
+    "--without-ndiff",
+    "--without-ncat",
+]
+configure_gen = []
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "gmake",
+]
+makedepends = [
+    "linux-headers",
+    "openssl-devel",
```

Comment:
you are missing a bunch of libraries and configure args to make it not use its own vendored libs

Reply:
perhaps

```
		--with-openssl=yes
		--with-libpcap=yes
		--with-libpcre=yes
		--with-libz=yes
		--with-libssh2=yes
		--with-liblua=/usr/lua5.4
```

with the respective libs

Reply:
you'll need to rebase

https://github.com/nmap/nmap/commit/828ab48764b82d0226e860c73c5dac5b11f77385
https://github.com/nmap/nmap/commit/4c0ed54f6848ddf1222c7d4828dc2d652c2f5f72
https://github.com/nmap/nmap/commit/c6ffdbf7ae35e397b663f787fc4934f01458f945

for pcre2 to work

Reply:
hm, interestingly it links it anyway

Reply:
i still think you need parts of those commits though

Reply:
I'm sorry I'm not sure what this means. Would this mean applying the changes in those commits via patch files to the source after downloading?

Reply:
you would put patches in a folder called patches/ and they get applied for you, yeah

Reply:
From what I can tell

https://github.com/nmap/nmap/commit/828ab48764b82d0226e860c73c5dac5b11f77385 - This is already included in release 7.95
https://github.com/nmap/nmap/commit/4c0ed54f6848ddf1222c7d4828dc2d652c2f5f72 - The bad patterns don't exist in 7.95
https://github.com/nmap/nmap/commit/c6ffdbf7ae35e397b663f787fc4934f01458f945 - I've now included a patch for this with the exception of the changes to `configure.ac` (autoheader breaks if they were included)

Reply:
thanks!